### PR TITLE
- first attempt at testing

### PR DESCRIPTION
**Pull request by yfarjoun**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CircleCI 2.0 configuration file
+#
+#
+version: 1
```
Comment (kachulis) [inline .circleci/config.yml]: can we use circleci 2.0 instead?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: broadinstitute/womtool:47-b36d920
+        entrypoint: /bin/bash
+    branchs:
```
Comment (kachulis) [inline .circleci/config.yml]: branches

Comment (yfarjoun) [reply .circleci/config.yml]: (I guess it didn't do anything...)

File: .circleci/config.yml (hunk 1 above)
Comment (yfarjoun) [reply .circleci/config.yml]: 👍

**Pull request merged**